vispy/app/application.py

```diff
@@ -77,6 +77,19 @@ def process_events(self):
         """
         return self._backend._vispy_process_events()
 
+    def sleep(self, duration_sec):
+        """
+        Sleep for the given duration. This is used to reduce
+        CPU stress when VisPy is run in interactive mode.
+        see inputhook.py for details
```

Comment:
check out the `numpy` docstring spec -- this should be a one-line description on the same line as the `"""`, followed by more lines later if need be:

https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections